openapi.yaml

```diff
@@ -7,8 +7,8 @@ info:
     email: team@redocly.com
     url: 'https://redocly.com/docs/cli/'
   license: 
-    name: Apache 2.0
-    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+    name: MIT
+    url: 'https://www.tldrlegal.com/license/mit-license'
```

Comment:
Could we use a less commercially-oriented site to link to as the source? I think https://opensource.org/license/mit/ would be more appropriate.